lblod/frontend-form-builder
# SFB-246: make the concepts draggable to set an order

## ID
 [SFB-246](https://binnenland.atlassian.net/browse/SFB-246)

 ## Description

Now that the concept model has a order predicate we can make it possible for the user to change the order of the concepts. We will make the rows of the table draggable so the user can set each concept in the correct order.

 ## Type of change


 ## How to test

1. Open a codelist check the order
2. When you have a custom codelist you are able to change the order
3. The data should be updated with the correct order value when it has been dragged
4. Save the changes and see if it is saved

 ## Links to other PR's

 - #161 

## Attachments

![image](https://github.com/lblod/frontend-form-builder/assets/36266973/af5ee5f2-bdb2-43c0-9700-b946d37029b3)
